src/components/menu/menu.component.tsx

```diff
@@ -80,5 +80,4 @@ export const Menu = ({ menuType = "light", children, ...rest }: MenuProps) => {
   );
 };
 
-export { MenuContext };
```

Comment:
comment: good spot, annoyingly I missed this when I moved all the contexts to internal folders. Unfortunately this will need to be a breaking change as it's exported as part of the public facing components, so lets not remove it from this file at this time and batch it with other trivial breaking changes. We can leave the path updates that you've made and then remove this when we have some more housekeeping to do